extensions/interactions/FractionInput/directives/oppia-interactive-fraction-input.component.spec.ts

```diff
@@ -131,14 +131,15 @@ describe('InteractiveFractionInputComponent', () => {
     component.answerValueChanged();
     component.answer = '12345678';
 
-    expect(component.errorMessage).toBe('');
+    expect(component.errorMessageI18NKey).toBe('');
```

Comment:
Just a minor note, it should technically be errorMessageI18nKey following our usual lowerCamelCase conventions (note the lowercase n in I18n). Ditto elsewhere.

Reply:
Done